test/serializer_test.exs

```diff
@@ -1,23 +1,24 @@
 defmodule JSONAPISerializerTest do
-  use ExUnit.Case
+  use ExUnit.Case, async: false
```

Comment:
It's a shame we have to make this non-async.  I'm wondering if that suggests we should re-think this approach a bit?

Reply:
It's required because of the environment modification.  Best option is probably an argument to one of the functions, but I'm not sure which is the right option.

Reply:
The tests are fast enough that its largely a non-issue anyways :P

Reply:
I have no doubt they are!  I just always stop to reconsider the approach when tests starting have to be sequential, tends to indicate some type of smell (at least in my experience).

Reply:
@doomspork do you think an `opts` arg should be added to `JSONAPI.Serializer.serialize`?

Reply:
That's a great question/idea @mitchellhenke!  I haven't thought about it. 
 Do you think it would make it more difficult for folks to leverage the library and modules?

I don't think there's too many cases where folks use `serialize/3` directly, do you?

On a side note: I need the 🤔  for a reaction!

Reply:
@doomspork I'm not sure.  The `conn` argument could potentially go in an `opts` as well, but it changes the API of anyone using it directly.